chrome-launcher/.npmignore

```diff
@@ -0,0 +1,20 @@
+
+# folders
+.vscode/
+test/
+node_modules/
+
+# include all compiled JS
+!*.js
```

Comment:
gitignore 
https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package